### DISCUSSION TITLE
How to deploy AutoUpdate settings without Intune?

### DISCUSSION THREAD

**Discussion by JasonCioffi** [Romanitho/Winget-AutoUpdate#600] [Q&A] [answered]
Hi,

I downloaded the admx file with the intention of installing it on my on-premise AD DC to push settings to WAU and I see it's not formatted for that.  Is there an admx file that is formatted for on-premise AD DC?  Or is there another way I can push settings to client PC's?

Thanks in advance for your replies.

**Comment by KnifMelti** [comment]
Strange. No problem in our on-premise AD.
2019 servers, latest schematics

**Comment by JasonCioffi** [comment, marked as answer]
I downloaded the html from Github, rather than the code from the GPO's themselves.  I figured it out.
